Sources/ParseSwift/Types/Query.swift

```diff
@@ -24,6 +30,15 @@ extension Querying {
     func count() throws -> Int {
         return try count(options: [])
     }
+    func find(completion: @escaping (Result<[ResultType], ParseError>) -> Void) {
```

Comment:
Should this method (and also `first` and `count`) take a callback queue instead of hardcoding it to `.main`?

Reply:
I’ll verify this later, but If I remember correctly, these extensions are setup to create the default .main you mentioned earlier

Reply:
These are setup correctly as there's another extension in the file on Line 212 that allows for both `options` and `callbackQueue` to be used. The ones you referenced are to make it easier for the user as you pointed out earlier